RedHatInsights/ros-frontend
# RHIROS-906 updated packages to fix local setup and vulnerabilities

### Description

Few of the packages were old and hence were creating problem while running `npm install` command. I have updated the package to fix the issue and also it fixes few of the vulnerabilities.

I have created a card in backlog to prioritise and fix rest of the vulnerabilities.
https://issues.redhat.com/browse/RHIROS-904

### Steps to test

* Checkout the branch
* Remove `node modules` folder
* Run `npm install` command, it should install the mentioned packages successfully
* Run `npm start` command and check if frontend is working properly.
*
